kubescaler/scaler/aws/cluster.py

```diff
@@ -668,6 +668,25 @@ def scale(self, count):
             ],
         )
 
+        # Wait for stack update to be complete. 
```

Comment:
Great work here! My one suggestion is to make this a function that other potential waiting stacks could use.

Reply:
sure. that's a good idea. I'll make the function and use that function in all waiting cases.